### PR TITLE
made all commands non case sensitive

### PR DESCRIPTION
**Pull request by Bandov**
All commands now non case sensitive

### CODE REVIEW

**Comment by Bandov** [comment]
also added the functionality of shorthand. Each shorthand is just the first letter of each command. It means EditRelation == er and addAttribute == aa

**Review by Emberlynn-Loo — APPROVED** [review]
LGTM

**Comment by DingBao-sys** [comment]
LGTM

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -71,51 +71,67 @@ public Command parseCommand(String userInput) throws ParseException, CommandExce
...
 
         case HelpCommand.COMMAND_WORD:
+        case HelpCommand.COMMAND_WORD_SHORT:
             return new HelpCommand();
 
         case AddRelationshipCommand.COMMAND_WORD:
+        case AddRelationshipCommand.COMMAND_WORD_SHORT:
             return new AddRelationshipCommandParser().parse(arguments.trim());
 
         case DeleteRelationshipCommand.COMMAND_WORD:
```
Comment (DingBao-sys) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:105]: I liked how you use case fall through pretty smart

Comment (Bandov) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java:105]: thanks ik i am

**Pull request merged**